pre-docs/control_repo.md

```diff
@@ -0,0 +1,21 @@
+# The control repository pattern
+
+The control repository pattern is useful for creating a project that contains your tasks and plans - along with declaring modules they depend on - for others to run. It's based on the idea originally [developed for Puppet](https://github.com/puppetlabs/best-practices/blob/master/control-repo-contents.md) of a repository that acts as a single location for an organization's configuration. The base example can be found [here](https://github.com/puppetlabs/control-repo).
+
+In Bolt, the control repository can also act as the `Boltdir`. To do so, add `bolt.yaml` (it can be an empty file) to the root of the repository.
```

Comment:
```suggestion
In Bolt, the control repository can also act as the [Boltdir](https://puppet.com/docs/bolt/1.x/configuring_bolt.html). To do so, add `bolt.yaml` (it can be an empty file) to the root of the repository.
```

Reply:
That was the best place I could find explaining what a boltdir is - should we give it it's own page or include a more full explanation of what it is?